app/routes/articles.js

```diff
@@ -77,13 +79,13 @@ router.get('/approve', isLoggedIn, (req, res) => {
 })
 
 //APPROVE Show Article Route
-router.get('/approve/:id', isLoggedIn, async (req, res) => {         
-    if(!req.user.isAdmin || !req.params.id === undefined) {
+router.get('/approve/:title', isLoggedIn, async (req, res) => {         
+    if(!req.user.isAdmin || !req.params.title === undefined) {
```

Comment:
`if(!req.user.isAdmin || !req.params.title)`
would also check if title is an empty string